lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -91,60 +90,68 @@ export default function BasicLtiLaunchApp({ rpcServer }) {
    *  "error-authorizing", or "error-report-submission"
    * @param {boolean} [retry=true] - Can the request be retried?
    */
-  const handleError = (e, newErrorState, retry = true) => {
+  const handleError = useCallback((e, newErrorState, retry = true) => {
```

Comment:
I don't think the change to `handleError` is strictly necessary here, although it doesn't do any harm to use `useCallback`. When I tested on the `master` branch, the linting seemed smart enough to realize that it doesn't matter if the `handleError` callback was not a dependency because it doesn't read any state.